DayrSuryan/DSEthiop8.xml

```diff
@@ -715,7 +715,7 @@ The Coptic patriarch <persName ref="PRS3258CyrilIV">Cyril IV (in tenure <date>18
                           <roleName type="title"><hi rendition="simple:italic">ʾAbbā</hi></roleName> Salāmā (in tenure <date>1841–1867</date>)</persName> are mentioned in a supplication
                        on <locus target="#6r"/>. The name of
                        <persName ref="PRS11944GabraM">Gabra Māryām</persName>, probably the commissioner, is mentioned
-                       in the supplications (eg., <locus target="#2r"/>, <locus target="#6v"/>). The manuscript
+                       in the supplications on <locus target="#2r"/> and <locus target="#6v"/>. The manuscript
```

Comment:
I feel that this modification changes the sense. The first version implies that there are more than two supplications, the modification that these are the only ones or the only ones in which he is mentioned. Is this the case?

Reply:
modified